src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2785,6 +2802,8 @@ namespace PurchasedAirManager {
         PurchAir(PurchAirNum).OutdoorAirVolFlowRateStdRho = OAMassFlowRate / StdRhoAir;
         PurchAir(PurchAirNum).SupplyAirMassFlowRate = SupplyMassFlowRate;
         PurchAir(PurchAirNum).SupplyAirVolFlowRateStdRho = SupplyMassFlowRate / StdRhoAir;
+        PurchAir(PurchAirNum).ZoneIdealLoadsSupplyAirTemperature = Node(InNodeNum).Temp;
```

Comment:
This is just passing the inlet temp and hum-rat through. I believe we want to use the already existing local variables ```SupplyTemp``` and ```SupplyHumRat``` to set these values.

Reply:
Or would it work to just move `SupplyTemp` and `SupplyHumRat` from local into the `PurchAir` struct?

Reply:
And while you're in here, `MixedAirTemp` and `MixedAirHumRat` would also be useful outputs.

Reply:
That's fine with me.

Reply:
Will do!